paper-button-behavior.html

```diff
@@ -14,35 +14,42 @@
 <script>
 
   /** @polymerBehavior */
+  Polymer.PaperButtonBehaviorImpl = {
+
+    properties: {
+
+      _elevation: {
+        type: Number
+      }
+
+    },
+
+    observers: [
+      '_calculateElevation(focused, disabled, active, pressed, receivedFocusFromKeyboard)'
+    ],
+
+    hostAttributes: {
+      role: 'button',
+      tabindex: '0'
+    },
+
+    _calculateElevation: function() {
+      var e = 1;
+      if (this.disabled) { // Note: or !raised
```

Comment:
hmm, why or !raised? won't a `<paper-button>` without raised, not active and not focused get an elevation of 1? or is this just never called in that case?